examples/example-deno-runtime/tests.ts

```diff
@@ -215,7 +215,7 @@ const imports: Imports = {
       url: "https://fiberplane.dev/",
       method: "POST",
       headers: {
-        "Content-Type": "application/json",
+        "content-type": encoder.encode("application/json"),
```

Comment:
HTTP headers are case-insensitive, and the `http` crate converts them to lowercase.